File: src/main/java/thedarkcolour/futuremc/enchantment/EnchantmentCrossbowBase.java
package thedarkcolour.futuremc.enchantment;

import net.minecraft.enchantment.Enchantment;
import net.minecraft.inventory.EntityEquipmentSlot;

public class EnchantmentCrossbowBase extends Enchantment{
    public EnchantmentCrossbowBase(Enchantment.Rarity rarity, String name) {
        super(rarity, Enchantments.CROSSBOW, new EntityEquipmentSlot[] {EntityEquipmentSlot.MAINHAND});
        setRegistryName(name);
        setName(name);
    }
}

File: src/main/java/thedarkcolour/futuremc/enchantment/EnchantmentCrossbowBase.kt
package thedarkcolour.futuremc.enchantment

import net.minecraft.enchantment.Enchantment
import net.minecraft.inventory.EntityEquipmentSlot
import thedarkcolour.futuremc.registry.FEnchantments

open class EnchantmentCrossbowBase(rarity: Rarity, name: String) : Enchantment(
    rarity,
    FEnchantments.CROSSBOW,
    arrayOf(EntityEquipmentSlot.MAINHAND)
) {
    init {
        setRegistryName(name)
        setName(name)
    }
}